UTM-Robotics/Deerhunt-Infrastructure
# Alex teams

A small PR for the creation of teams as well as some tests written for it.
To test:
1. run the server on one terminal.
2. run $ pytest -v test_teams.py

should pass.
